source/transactions/tests/retryable-commit-errorLabels.yml

```diff
@@ -0,0 +1,133 @@
+runOn:
+    -
+        minServerVersion: "4.3.x"
```

Comment:
We can't actually use ".x" here (or at least that isn't specified). It needs to be a real version number. Since we know 4.3.0 doesn't have the functionality we need, you may want to ask the server team if they expect to have the changes implemented for the next patch release. If so, you can preemptively put 4.3.1.

Reply:
Yes! To be clear, I'm just putting that as a placeholder until I know the actual server number for easy search and replace.